.env.example

```diff
@@ -58,4 +58,7 @@ TRANSACTIONS_RETRY_OFFSET=60
 CLOSURE_RETRY_INTERVAL=30
 
 JWT_SECRET=ODMzNUZBNTZENDg3NTYyREUyNDhGNDdCRUZDNzI3NDMzMzQwNTFEREZGQ0MyQzA5Mjc1RjY2NTQ1NDk5MDMxNzU5NDc0NUVFMTdDMDhGNzk4Q0Q3RENFMEJBODE1NURDREExNEY2Mzk4QzFEMTU0NTExNjUyMEExMzMwMTdDMDk
-JWT_TOKEN_VALIDITY_TIME_MILLIS=600000
\ No newline at end of file
+JWT_TOKEN_VALIDITY_TIME_MILLIS=600000
+
+CONFIDENTIAL_DATA_MANAGER_KEY=EWsfdHUIDwLwYwC0zIMlrw==
+CONFIDENTIAL_DATA_MANAGER_ALGORITHM=AES
```

Comment:
This should **always** be AES, as currently it's the only cipher `ConfidentialDataManager` understands. Maybe it warrants a change of interface in `ecommerce-commons`, what do you think?

Reply:
Yes, if we are not planning to change encryption algorithm I think that this parameter can be removed altogether

Reply:
Fixed into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/150/commits/c0ff765bde5bc8d25a69ff0d7e92d77dae01ef6f